api/HTMLElement.json

```
@@ -868,6 +868,67 @@
           }
         }
       },
+      "enterKeyHint": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
+            },
+            "chrome_android": {
+              "version_added": "77"
+            },
+            "edge": {
+              "version_added": false
```

```suggestion
              "version_added": "79"
```

Based on testing https://mdn-bcd-collector.appspot.com/tests/api/HTMLElement/enterKeyHint in Edge 18 and 79.